apps/client/src/widgets/community/components/filter-dropdown/filter-dropdown.css.ts

```diff
@@ -24,6 +24,14 @@ export const DropDownIcon = style({
   transition: 'transform 0.3s ease',
 });
 
+export const iconTransparent = style({
+  backgroundColor: 'transparent',
+});
+
+export const iconWhiteBackground = style({
+  backgroundColor: themeVars.color.whiteBackground,
+});
+
```

Comment:
요렇게요! 추가로 밑에 IsRotate 를 지워야 해요.

```suggestion
export const DropDownIcon = recipe({
  base: {
    transition: 'transform 0.3s ease',
  },
  variants: {
    background: {
      transparent: {
        backgroundColor: 'transparent',
      },
      whiteBackground: {
        backgroundColor: themeVars.color.whiteBackground,
      },
    },
    isRotate: {
      true: {
        transform: 'rotate(180deg)',
      },
      false: {
        transform: 'rotate(0deg)',
      },
    },
  },
  defaultVariants: {
    background: 'transparent',
    isRotate: false,
  },
});
```